.circleci/create_circleci_config.py

```diff
@@ -98,11 +102,52 @@ def to_dict(self):
         pytest_flags.append(
             f"--make-reports={self.name}" if "examples" in self.name else f"--make-reports=tests_{self.name}"
         )
+
+        # We need explicit list instead of `pipeline.parameters.tests_to_run` (only available at job runtime)
+        tests = self.tests_to_run
+        if tests is None:
+            folder = os.environ["test_preparation_dir"]
+            test_file = os.path.join(folder, "filtered_test_list.txt")
+            if os.path.exists(test_file):
+                with open(test_file) as f:
+                    tests = f.read().split(" ")
```

Comment:
Not the best way to get the test list here. As the job objects are created before the `__main__`, we don't have `folder` information at object init. time.